boards/ti/lp_mspm0g5187/lp_mspm0g5187.dts

```diff
@@ -0,0 +1,133 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/dts-v1/;
+
+#include <ti/mspm0/g/mspm0g5187.dtsi>
+#include <ti/mspm0/g/mspm0g51xx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/clock/mspm0_clock.h>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "TI LP_MSPM0G5117";
+	compatible = "ti,mspm0g5117";
+
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		sw0 = &btn0;
+		sw1 = &btn1;
+		uart0 = &unicomm0_uart;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &unicomm0_uart;
+		zephyr,shell-uart = &unicomm0_uart;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
```

Comment:
this file is non compliant with the dts requirements

Reply:
@d-philpot  check the compliance errors, there's a tool you can run to autoformat the dts file